models/app-models.js

```diff
@@ -36,14 +36,16 @@ exports.selectArticles = ( sort_by = "created_at", order = "DESC", topic) => {
 }
 
 exports.selectArticleById = async (article_id) => {
-    let artiQuery = "SELECT * FROM articles"
+    let artiQuery = `SELECT articles.*,COUNT(comment_id) AS comment_count FROM articles`
+    artiQuery += ` LEFT JOIN comments ON comments.article_id = articles.article_id`
```

Comment:
There is no conditional logic here so is there a reason we add the join on after?